src/content/language/constructors.md

```diff
@@ -51,12 +51,12 @@ To instantiate a class, use a generative constructor.
 <?code-excerpt "point_alt.dart (idiomatic-constructor)" plaster="none"?>
 ```dart
 class Point {
-  // Initializer list of variables and values
-  double x = 2.0;
-  double y = 2.0;
+  // Instance variables to hold the coordinates of the point.
+  double x;
+  double y;
 
-  // Generative constructor with initializing formal parameters:
-  Point(this.x, this.y);
+  // Generative constructor with optional positional parameters and default values.
+  Point([this.x = 2.0, this.y = 2.0]);
```

Comment:
What do you think about just dropping the default values and making these required positional parameters?

Optional positional parameters and specifying their default values seem orthogonal to this section on "generative constructors".

Reply:
Hey @parlough, I guess that could work too!

The idea behind keeping default values was to retain the intent of the original code while showcasing optional positional parameters. That said, making the parameters required positional would work just as well.